src/imitation/algorithms/density.py

```diff
@@ -147,7 +147,8 @@ def _get_demo_from_batch(
         assert act_b.shape[1:] == self.venv.action_space.shape
 
         if isinstance(obs_b, types.DictObs):
-            exp_shape = {k: v.shape for k, v in self.venv.observation_space.items()}
+            exp_shape = {k: v.shape for k, v in self.venv.observation_space.items()}  # type: ignore[attr-defined] # noqa: E501
```

Comment:
We try to minimize number of errors we suppress -- can we reformat the code rather than suppressing the line length error?